Game/Source/ThirdPersonShooter/BigGym/BigGymGameMode.cpp

```diff
@@ -17,8 +20,142 @@ ABigGymGameMode::ABigGymGameMode()
 	if (PlayerPawnBPClass.Class != NULL)
 	{
 		DefaultPawnClass = PlayerPawnBPClass.Class;
+		PlayerCheckoutRadius = FGenericPlatformMath::Sqrt(PlayerPawnBPClass.Class.GetDefaultObject()->NetCullDistanceSquared);
 	}
-	
+
+	PlayersSpawned = 0;
+
 	// Seamless Travel is not currently supported in SpatialOS [UNR-897]
 	bUseSeamlessTravel = false;
 }
+
+void ABigGymGameMode::InitGameState()
+{
+	if (ShouldUseCustomSpawning())
+	{
+		ParsePassedValues();
+		ClearExistingSpawnPoints();
+		GenerateSpawnPoints(TotalPlayers);
+	}
+}
+
+bool ABigGymGameMode::ShouldUseCustomSpawning()
+{
+	FString WorkerValue;
+	USpatialWorkerFlags::GetWorkerFlag(TEXT("override_spawning"), WorkerValue);
+	return (WorkerValue.Equals(TEXT("true"), ESearchCase::IgnoreCase) || FParse::Param(FCommandLine::Get(), TEXT("OverrideSpawning")));
+}
+
+void ABigGymGameMode::ParsePassedValues()
+{
+	if (FParse::Param(FCommandLine::Get(), TEXT("OverrideSpawning")))
+	{
+		if (FParse::Param(FCommandLine::Get(), TEXT("TotalPlayers=")))
```

Comment:
Worth adding a log that your overriding the worker_flag setting here.

Reply:
👍